docs/variants/dell_optiplex/recovery.md

```diff
@@ -1,85 +1,211 @@
+open-source firmware. Following procedure is supported for following models
+
+<center>
+
+| Vendor | Model |
+:-------:|:-----:|
+|Dell    | OptiPlex 7010 SFF |
+|Dell    | OptiPlex 7010 DT |
+|Dell    | OptiPlex 9010 SFF |
 
-## **Prequisitions**
+</center>
 
-1. To proceed with recovery `.rom` file with backup vendor firmware will be
-    neccesary eg. `backup.rom`.
+## Step 1: Prepare hardware
 
-    Backup file should be generated before making any changes in device flash
-    chip according to documentation in
-    [BIOS backup](https://docs.dasharo.com/variants/dell_optiplex/initial-deployment/#bios-backup)
-    section.
+1. To prepare hardware please follow
+[hardware preparation](../initial-deployment/#hardware-preparation) procedure.
+2. Follow instruction in the video to remove heatsink:
+  <center>
+  <iframe width="640" height="480"
+    src="http://www.youtube.com/embed/TiUSTo-XwPo">
+  </iframe>
+  </center>
+## Step 2: Find SPI_1, SPI_2/SPI_3
 
-1. Set jumper according to
-    [hardware preparation](https://docs.dasharo.com/variants/dell_optiplex/initial-deployment/#hardware-preparation)
+<center>
+![](../../images/dell_optiplex_9010_spi_pin1.jpg)
+</center>
 
-1. Install flashrom using
-    [flashrom installation guide](https://docs.dasharo.com/variants/dell_optiplex/initial-deployment/#flashrom-installation)
+## Step 3: Connect SOIC-8 Pomona clip between RTE and target
 
----
+Connect SOIC-8 Pomona according to
+[MX25L3206E datasheet](https://www.macronix.com/Lists/Datasheet/Attachments/8616/MX25L3206E,%203V,%2032Mb,%20v1.5.pdf).
 
-## **Internal flashing**
+<center>
+![](../../images/mx25l3206e_pinout.png)
+</center>
 
-Flash firmware on you Dell OptiPlex machine using below command:
+<center>
+ ![Pomona SOIC clip](../../images/pomona_clip.jpg)
+</center>
+
+<center>
+
+ | RTE J7                                 | Pomona SOIC clip  |
+ |:--------------------------------------:|:-----------------:|
+ | CS                                     | pin 1 (upside)    |
+ | MISO                                   | pin 2 (upside)    |
+ | GND                                    | pin 4 (upside)    |
+ | VCC                                    | pin 5 (downside)  |
+ | SCLK                                   | pin 7 (downside)  |
+ | MOSI                                   | pin 8 (downside)  |
+
+</center>
+
+Numbers 1-4 have to be on one side and numbers 5-8 have to be on the other side
+of the clip.
+
+<center>
+![Clip up](../../images/clip_upside.jpg)
+</center>
+
+<center>
+![Clip down](../../images/clip_downside.jpg)
+</center>
+
+Clip on the `SPI_1` chip. Match pin 1 (`CS`) of the Pomona clip with the first
+pin of `SPI_1` chip, marked with a small dot engraved on the chip.
+
+![Clip connected](../../images/clip_connected.jpg)
+
+## Step 4: Connect RTE
+
+<center>
+![](../../images/rte-boot.jpg)
+</center>
+
+1. Connect J2 Orange Pi Zero system debug output
+2. Power the board and confirm it boots
```

Comment:
Whether all points should not be declared as `1`? If the text formatting is correct, these `1` should be converted into consecutive numbers.

Reply:
Fixed https://github.com/Dasharo/docs/pull/139/commits/1040d116a5113f6ed8ec300850b2b1b8441499f4